views/default/input/tel.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Elgg tel input
+ * Displays a tel input field
+ *
+ * @package Elgg
+ * @subpackage Core
+ *
+ */
+
+$vars['class'] = elgg_extract_class($vars, 'elgg-input-text');
+
+$defaults = [
+	'value' => '',
+	'disabled' => false,
+	'type' => 'tel',
+	'pattern' => '[0-9]{3}-[0-9]{2}-[0-9]{3}'	// Format: 123-45-678
```

Comment:
please don't add a default pattern, because it doesn't work for every country. 

Eg. for us in The Netherlands it should be `[0-9]{3,4}-[0-9]{6,7}`